hydra-cluster/src/CardanoCluster.hs

```diff
@@ -1,9 +1,9 @@
 {-# LANGUAGE DeriveAnyClass #-}
 {-# LANGUAGE DerivingStrategies #-}
```

Comment:
comment: Perhaps we should lift those in the `.cabal` file? Seems like they are "safe" and uncontroversial extensions to use everywhere